src/main/java/com/minecolonies/apiimp/initializer/InteractionValidatorInitializer.java

```diff
@@ -291,5 +291,11 @@ public static void init()
         InteractionValidatorRegistry.registerStandardPredicate(new TranslationTextComponent(COM_MINECOLONIES_COREMOD_BEEKEEPER_NOFLOWERS),
           citizen -> citizen.getWorkBuilding() instanceof BuildingBeekeeper
                        && ((BuildingBeekeeper) citizen.getWorkBuilding()).getModuleMatching(ItemListModule.class, m -> m.getId().equals(BUILDING_FLOWER_LIST)).getList().isEmpty());
+
+        InteractionValidatorRegistry.registerStandardPredicate(new TranslationTextComponent(COM_MINECOLONIES_COREMOD_ENTITY_CITIZEN_RAINING),
+          citizen -> citizen.getEntity().isPresent() && citizen.getEntity().get().getEntityWorld().isRaining());
```

Comment:
this should also have a !raid check, else shipit

Reply:
Wouldn't that cancel the interaction of a rain gets started after the citizen already complains about the rain?

Reply:
Yes, this will cancel the rain interaction of the citizen the moment a raid started.

Reply:
I will look into always triggering the raid interaction on raid begin and adding this then.

Reply:
This change had a bigger impact then I expected.
Its done now :)